docs/topic-guides/blendable-web-apps/client-side-stack.md

```diff
@@ -20,29 +20,51 @@ The implementation of JSON-Patch is provided by `json-patch-duplex.js`, which co
+To render HTML templates in the DOM we recommend the [`dom-bind`](https://www.polymer-project.org/2.0/docs/devguide/templates) custom element that is part of [Polymer](https://www.polymer-project.org/). It supports binding annotations using curly braces \(`{{}}`\), conditional fragments with [`dom-if`](https://www.polymer-project.org/2.0/docs/devguide/templates#dom-if) and loops with [`dom-repeat`](https://www.polymer-project.org/2.0/docs/devguide/templates#dom-repeat).
 
-The solution is based on the Web Components APIs from the latest  HTML specifications. It's approved by all major browser vendors and is [widely adopted](https://www.webcomponents.org/). However, not everything is there yet, some browsers require the `webcomponents.js` polyfill to be loaded to support for these APIs.
+The solution is based on the Web Components APIs from the latest  HTML specifications. It's approved by all major browser vendors and is [widely adopted](https://caniuse.com/#search=web%20components). However, not everything is there yet, some browsers require the `webcomponents-lite.js` polyfill to be loaded to support for these APIs.
 
 ### Template loading
 
-To build modular apps out of many partial views, use the `imported-template` Custom Element. It loads the templates from separate HTML files and stamp them to the DOM, it plays well with Polymer's data-bindings and regular DOM APIs.
-
-To make it easier to use with Starcounter, `imported-template` is wrapped in the [starcounter-include](https://github.com/Starcounter/starcounter-include) Custom Element, which also sets up the data binding between Palindrom and imported-template.
+To build modular apps out of many partial views, use the [starcounter-include](https://github.com/Starcounter/starcounter-include) custom element. It loads the templates from separate HTML files, stamps them to the DOM and binds the view-model as exposed by Palindrom. It plays well with Polymer's data-bindings and regular DOM APIs.
 
 ## The stack, visualized
 
 The following chart shows the bird-eye's view on the client side libraries used.
 
 ![](../../.gitbook/assets/client-side-components.svg)
 
-## Versions
-
-See the file `ClientFiles\bower-list.txt` in your Starcounter installation directory \(usually `C:\Program Files\Starcounter`\) for the list of client side libraries bundled with your Starcounter instance, including their version numbers.
-
+## Client side libraries distributed with Starcounter
+
+The `StaticFiles` folder from Starcounter installation is automatically served as a static content folder. When Starcounter server receives a request for a static file, it searches for the file in all of the static content folders. The project folder has higher priority over internal folder.
```

Comment:
The project folder has higher priority over internal folder. -> The project folder has a higher priority than the internal folder.